src/libraries/Microsoft.PowerFx.Transport.Attributes/Microsoft.PowerFx.Transport.Attributes.csproj

```diff
@@ -19,5 +19,12 @@
     <Title>Microsoft PowerFx Transport Attributes</Title>
     <Description>This provides attributes used by Microsoft.PowerFx.Core</Description>
     <Summary>This provides attributes used by Microsoft.PowerFx.Core</Summary>
+    <PackageReadmeFile>readme.md</PackageReadmeFile>
   </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="..\readme.md" Link="readme.md" Pack="true" PackagePath="\">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```

Comment:
When somebody includes multiple of our nugets, will these readmes collide with each other?

Reply:
nugets are usually stored %userprofile%\.nuget\packages and have their own folder